### ISSUE TITLE
Documentation website isn't working

### ISSUE THREAD

**Issue by Snailedlt** [dylanblokhuis/svelte-feather-icons#18]
![image](https://user-images.githubusercontent.com/43886029/203111279-f634197e-3f9b-4012-9486-7220688cfd32.png)

**Comment by webdagger** [comment]
Yes, I have this issue as well. I wonder if @dylanblokhuis we can host a simple version on Vercel.
